index.js

```diff
@@ -63,7 +63,30 @@ module.exports = {
 
     if (isProductionBuild || isOldEmber) {
       tree = new Funnel(tree, {
-        exclude: [/instance-initializers\/axe-component|violations-helper.js/]
+        exclude: [/instance-initializers\/(axe-component|violations-helper)\.js/]
+      });
+    }
+
+    return tree;
+  },
+
+  /**
+   * Exclude all addon code during build if this is a
+   * production build or if the version of Ember being used is less than 1.13.
+   * @override
+   */
+  treeForAddon: function() {
+    var tree = this._super.treeForAddon.apply(this, arguments);
+    var checker = new VersionChecker(this);
+    var isProductionBuild = process.env.EMBER_ENV === 'production';
+    var isOldEmber = checker.for('ember', 'bower').lt('1.13.0');
```

Comment:
There is a special check added in ember-cli-version-checker for Ember specifically, so you don't have to specify bower vs npm:

```suggestion
    var isOldEmber = checker.forEmber().lt('1.13.0');
```

Reply:
Thanks for the suggestion @rwjblue. I'll get this updated.